predicators/ml_models.py

```diff
@@ -1041,7 +1135,8 @@ def _train_pytorch_model(model: nn.Module,
                          batch_generator: Iterator[Tuple[Tensor, Tensor]],
                          max_train_iters: MaxTrainIters,
                          dataset_size: int,
-                         print_every: int = 1000,
+                         device: torch.device,
+                         print_every: int = 10,
```

Comment:
can we make this a setting that gets passed through? and then make sure that when you run the command below before and after your change, the printouts and time taken look about the same:

`python predicators/main.py --approach nsrt_learning --env cover --seed 0`